Locator/FailureTransportLocator.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Locator;
+
+use Symfony\Component\Messenger\Transport\TransportInterface;
+use Symfony\Contracts\Service\ServiceProviderInterface;
+
+class FailureTransportLocator
```

Comment:
i'm wondering if this should not be merged into `ReceiverLocator`

Reply:
I'm thinking that too. There is also a PR (that I support in principle) inside Symfony to allow for multiple failure transports - basically on per normal transport. If we want to future-proof a little bit, instead of other classes being able to call `$failureTransportLocator->getFailureTransport()`, we're going to need to keep track of which "failure transport" we're working with and pass that to methods. For example, in `FailedMessageRejecter::rejectFailedMessage()`, we might need a `$failureReceiveName` as an argument. 

If we did that, indeed, we would *only* need the `ReceiverLocator`. Because now code like `FailedMessageRejecter` would be calling something like `$this->receiverLocator->getReceiver($failureReceiveName)` to get the failure transport.